frontend/src/app/AppRouter.tsx

```diff
@@ -111,10 +111,9 @@ export const AppRouter = (): JSX.Element => {
           element={<PrivateElement element={<AdminFormLayout />} />}
         >
           <Route index element={<CreatePage />} />
-          <Route
-            path={ADMINFORM_SETTINGS_SUBROUTE}
-            element={<SettingsPage />}
-          />
+          <Route path={ADMINFORM_SETTINGS_SUBROUTE} element={<SettingsPage />}>
+            <Route path={':settingsTab'} element={<SettingsPage />} />
```

Comment:
With this it means any `settings/:settingsTab` is a new path and has the same functionality + interaction as `settings/` right?

Just a slight concern cause technically after getting redirected back, such as from payments connect, when you toggle around the settings page, the `settings/payments` will still be reflected in the url

Was wondering if there are ways to just capture the param, maybe after computing tabIndex, navigate back to `settings/`?  (Or use search params instead?)

Minor thing though, not sure if its significant

Reply:
hmmm okay true... might be useful to sort this out especially if we want to do metrics 🤔

Reply:
Wondering if its better to somehow bind the change in tabs to the path, or use query params as you suggested

Reply:
Yeah so either by binding so the `/:settingsTab` reflects the current tab selected, and any erroneous path will lead back to say `settings/general` 

Else if query params we can do like `settings?tab=payments` then after computing `tabIndex` we can do a 
'''
const [searchParams, setSearchParams] = useSearchParams()
searchParams.delete('tab')
setSearchParams(searchParams)
'''
I think should be able to modify the url string back to `/settings`?

Reply:
Decided to go with the path route so it'll be easier to do metrics if we want to do user tracking in the future. Bound the tab changes to the paths, but I can't bind the default path to `/general` on navigation because it takes some time before `displayPayments` can be determined... I think that's okay though, right?

Reply:
Done in [b5f9761](https://github.com/opengovsg/FormSG/pull/6348/commits/b5f9761880ea389c261c41c67e1033fe39ef7498)!